buildconfig/stubs/pygame/surface.pyi

```diff
@@ -70,6 +70,13 @@ class Surface:
         ],
         doreturn: Union[int, bool] = 1,
     ) -> Union[List[Rect], None]: ...
+    def ublits(
+            self,
+            blit_sequence: Union[
+                Sequence[Tuple[Surface, Union[Coordinate, RectValue]]],
+                List[Tuple[Surface, Union[Coordinate, RectValue]]]],
+            blend_flags: int
```

Comment:
```suggestion
            blend_flags: int, 
            /,
```
This will mark that the function takes positional-only args

Reply:
Pushed commit to address the remaining issues

Reply:
This one does not seem to be resolved yet

Reply:
oh yeah i thought i put / but ended up only putting a , lol